tests/unit/nupic/research/sp_learn_inference_test.py

```diff
@@ -108,7 +109,8 @@ def _runLearnInference(self,
       if spVerbosity > 0:
         print "Input #%d" % i
       encodedInput = inputs[i]
-      spLearnInfer.compute(encodedInput, learn=True, infer=False)
+      decodedOutput = np.zeros(columnDimensions)
+      spLearnInfer.compute(encodedInput, learn=True, activeArray=decodedOutput)
```

Comment:
Can you put links in the code wherever there used to be `infer-False` so that it is possible to fix up all these cases after implementing the feature? Otherwise it would be next to impossible to find all the different places that aren't using the output.